azure-pipelines.yml

```diff
@@ -6,51 +6,26 @@ trigger:
 
 jobs:
   # Linux
-  - job: Linux
+  - job: 'Linux'
```

Comment:
It is not consistent with windows and mac jobs. Maybe add quote to the others too or remove the quote here?